pkg/cli/admin/upgrade/status/controlplane.go

```diff
@@ -288,6 +427,6 @@ const controlPlaneStatusTemplateRaw = `= Control Plane =
 Assessment:      {{ .Assessment }}
 Target Version:  {{ .TargetVersion }}
 Completion:      {{ printf "%.0f" .Completion }}%
-Duration:        {{ .Duration }}
+Duration:        {{ shortDuration .Duration }}{{ if .EstTimeToComplete }} (Est Time Remaining: {{ vagueUnder .EstTimeToComplete .EstDuration }}){{ end }}
```

Comment:
nit: can we use "Est. Time Remaining" or "estimated time remaining" or something that either has a `.` to say "I'm abbreviating!" or doesn't abbreviate?

Reply:
This comes from Justin's [mockup](https://docs.google.com/presentation/d/1BtVSGWaG6LD0qye9BHPU5ssnEYTGPOlhHeogFfECUnk/edit#slide=id.g25513f80e3d_0_514)... 

I don't mind adding a dot there too much but I liked the output without it more actually. The output is quite technical and reads like a dashboard, abbreviations without a dot are not that uncommon in structured technical content.

Reply:
Addressed in [503b0f5](https://github.com/openshift/oc/pull/1742/commits/503b0f5f9d7b20ecc871b87a16fdafe13338fd04)